File: CS/DXSample/Services/TabControlService.cs
using DevExpress.Mvvm.UI;
using DevExpress.Xpf.Core;

namespace DXSample.Services {
    public class TabControlService : ServiceBase, ITabControlService {
        DXTabControl AssociatedTabControl { get { return (DXTabControl)AssociatedObject; } }

        public void Hide(object item) {
            AssociatedTabControl.HideTabItem(item);
        }

        public void Show(object item) {
            AssociatedTabControl.ShowTabItem(item);
        }
    }
}

File: VB/DXSample/Services/TabControlService.vb
Imports DevExpress.Mvvm.UI
Imports DevExpress.Xpf.Core

Namespace DXSample.Services
    Public Class TabControlService
        Inherits ServiceBase
        Implements ITabControlService

        Private ReadOnly Property AssociatedTabControl() As DXTabControl
            Get
                Return CType(AssociatedObject, DXTabControl)
            End Get
        End Property

        Public Sub Hide(ByVal item As Object) Implements ITabControlService.Hide
            AssociatedTabControl.HideTabItem(item)
        End Sub

        Public Sub Show(ByVal item As Object) Implements ITabControlService.Show
            AssociatedTabControl.ShowTabItem(item)
        End Sub
    End Class
End Namespace
